### PR TITLE
Feature/viewmodel

### PR DESCRIPTION
**Pull request by luizeth0**
Added the class CocktailsAppViewModel.kt that has a ViewModel instance from the androidx.lifecycle library

### CODE REVIEW

File: app/src/main/java/com/example/thecocktailsapp/viewmodule/CocktailsAppViewModel.kt [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import com.example.thecocktailsapp.model.CocktailIInfo.CocktailsResponse
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+
+class CocktailsAppViewModel(
+    private val cocktailsRepository: CocktailsRepository
+): ViewModel() {
+
+    // Expose screen UI state
+    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO
```
Comment (inaki92) [inline app/src/main/java/com/example/thecocktailsapp/viewmodule/CocktailsAppViewModel.kt]: Lets have this in the constructor, in order to pass a test dispatcher when testing

**Review by inaki92 — CHANGES_REQUESTED** [review]

**Comment by inaki92** [comment]
@luizeth0 please add you unit testing

**Review by chr1s4ng3l — DISMISSED** [review]

**Review by sarisiza — DISMISSED** [review]
LGTM

**Review by chr1s4ng3l — APPROVED** [review]
lgtm

**Pull request closed without merging**